@navikt/core/react/src/form-summary/utils.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { HR } from "../layout/hr/HR";
+
+export const injectHRBetween = (children: React.ReactNode) => {
```

Comment:
Kan hende dette kan løses med litt enkel CSS. Skjermleser får allerede semantikk ved bruk av dl > dt dd liste, så trengs ikke eget HTML-element for seksjonering 🤔

Reply:
tanken var å forkenkle CSS her 🤔, i steden for å dele opp whitespace i 2 + border bottom/top? 

eller tenker du med `:before`?